### ISSUE TITLE
VoiceOver skips non-existent page 2 of 3

### ISSUE THREAD

**Issue by MikePulsiferDOL** [USDepartmentofLabor/Comply-Chain#229]
On the main screen, when swiping to the next set of steps, VoiceOver reads "page 3 of 3."  Swiping back, it reads "page 1 of 3."  This can leave visually impaired users with the belief that there is a page of content or menu options that are being skipped.

https://user-images.githubusercontent.com/1970829/169328492-ad57db40-1663-4fe7-b876-88d13143377b.mov

**Comment by pbhatt17** [comment]
https://user-images.githubusercontent.com/42007436/170296201-6652c013-4825-4eba-91a8-558031202b0f.MP4

**Comment by vikramkm114a** [comment]
Hi Mike, we tested this issue on multiple devices, using multiple prior versions of the build as well as the current application from the app store and were unable to reproduce your results. Please see the attached video for further information.
